app/src/main/res/values/strings.xml

```diff
@@ -204,6 +204,12 @@
     <string name="pref_desc_ui_article_fontSerif">Serif font for articles</string>
     <string name="pref_name_ui_article_textAlignment_justify">Text alignment: Justify</string>
     <string name="pref_desc_ui_article_textAlignment_justify">Stretches lines to equal width (like in newspapers)</string>
+    <string name="pref_name_ui_article_handlePreformattedText">Handling Long Preformatted Text</string>
+    <string name="pref_desc_ui_article_handlePreformattedText">How To handle preformatted text that is too long to show completely.</string>
+    <string-array name="pref_option_handlePreformattedText" translatable="true">
```

Comment:
`translatable="true"` is not needed here and in `values-de`.
Otherwise looks ok.

Reply:
I did the rebase and also applied your change.